biosimulators_utils/config.py

```diff
@@ -96,7 +106,9 @@ def __init__(self,
                  BIOSIMULATIONS_API_AUTH_ENDPOINT=DEFAULT_BIOSIMULATIONS_API_AUTH_ENDPOINT,
                  BIOSIMULATIONS_API_AUDIENCE=DEFAULT_BIOSIMULATIONS_API_AUDIENCE,
                  VERBOSE=False,
-                 DEBUG=False):
+                 DEBUG=False,
+                 STDOUT_LEVEL=DEFAULT_STDOUT_LEVEL,
+                 **CUSTOM_SETTINGS):
```

Comment:
using the `**` I see. kind of dangerous that anything can come in here and will be applied to the settings. Do you have examples of what kind of kwargs might be used?

Reply:
Devising some presentable use cases now to plead my case, hah.